build.cake

```diff
@@ -323,7 +323,7 @@ Task("OnlyPublish")
             }
         }
     }
-    CreateRunScript(System.IO.Path.Combine(publishFolder, project, "default"), scriptFolder);
+    CreateRunScript(System.IO.Path.Combine(publishFolder, project, "default"), scriptFolder, "OmniSharp");
```

Comment:
Introduce constant? I'd hate to see a build failure simply due to the file system being case sensitive.